frontend/src/app/components/sidebar/sidebar.component.scss

```diff
@@ -0,0 +1,10 @@
+::ng-deep .p-submenu-header {
```

Comment:
I would avoid overriding prime-ng classes as long as possible and rely on the default behavior

And IF we really have to override some classes something then we should add :host to only override them in the context of the sidebar and not globally. (::ng-deep styles will "breakout" of the view encapsulation https://blog.angular-university.io/angular-host-context/)

If you want to do global overrides i would suggest to put them in the styles.scss of the project